feat and bugfixes: update kb and add edit command to support multigroups (#4156)
operator-framework/operator-sdk: website/content/en/docs/building-operators/golang/crds-scope.md

To ensure that the CRD is always generated with `scope: Cluster`, add the marker 
`// +kubebuilder:resource:path=<resource>,scope=Cluster`, or if already present replace `scope={Namespaced -> Cluster}`, 
above the CRD's Go type definition in `api/<version>/<kind>_types.go` or `apis/<group>/<version>/<kind>_types.go` 

> if you are using the [multigroup][multigroup-kubebuilder-doc] layout. Note that the `<resource>` 

Reviewer: ```suggestion
if your project is [multi-group][multigroup-kubebuilder-doc]. Note that the `<resource>` 
```
